backend/apps/owasp/utils/notifications.py

```
@@ -0,0 +1,29 @@
+def publish_snapshot_notification(snapshot: Snapshot) -> None:
+    """Publish a notification for a published snapshot."""
+    try:
+        redis_conn = get_redis_connection("default")
+        stream_key = "owasp_notifications"
+        message = {
+            "type": "snapshot_published",
+            "snapshot_id": str(snapshot.id),
+            "timestamp": str(now().timestamp()),
+        }
+        redis_conn.xadd(stream_key, message)
+        print(f"Successfully added message to Redis stream '{stream_key}'")
```

Ruff (0.14.14)

[warning] `print` found

Remove `print`

(T201)